lib/dry/component/container.rb

```diff
@@ -123,6 +123,8 @@ def self.require(*paths)
       end
 
       def self.load_component(key)
+        return if key?(key)
```

Comment:
Or rather, document the meaningful value in a "something was actually loaded" case vs nil in "thing was already loaded" case :)